### ISSUE TITLE
npm 2.6.0 per-package engineStrict will no longer be used

### ISSUE THREAD

**Issue by nahody** [gulpjs/gulp#915]
Please update the per-package engineStrict to use the config setting `engine-strict` instead.
In upcoming versions of npm it will no longer be used!

**Comment by heikki** [comment]
https://github.com/gulpjs/gulp/commit/f5e2ade41c475a16bcdcb24c9da7ae1af3ca0c1f

**Comment by drawcard** [comment]
After upgrading NPM to v. 2.6.1 I am getting more specific warning messages:

```bash
npm WARN engineStrict Per-package engineStrict (found in package.json for gulp)
npm WARN engineStrict won't be used in npm 3+. Use the config setting `engine-strict` instead.
npm WARN engineStrict Per-package engineStrict (found in package.json for gulp-concat)
npm WARN engineStrict won't be used in npm 3+. Use the config setting `engine-strict` instead.
npm WARN engineStrict Per-package engineStrict (found in package.json for vinyl-fs)
npm WARN engineStrict won't be used in npm 3+. Use the config setting `engine-strict` instead.
Killed
```
then ```npm install``` dies with error code -137.

I think I might downgrade NPM for the time being.

**Comment by heikki** [comment]
I get the same warnings with 2.6.1 but install is succesful.

**Comment by drawcard** [comment]
I rolled back to NPM 2.5.0 and Node v0.10.32 and ```npm install``` seems to run properly again for my project. 

I think I was getting the kill signal because my Vagrant VM was running out of memory, so I increased that to 1024MB. I might give NPM 2.6.1 a shot again if I get the time.

**Comment by BabylonSix** [comment]
why was this closed, I'm on npm 2.7.0 and the same problem persists.
Is the solution to just stay on npm 2.5.0?

**Comment by drawcard** [comment]
The warnings were harmless for me, and the -137 error was due to my VM running out of memory. Was that what you're referring to @BabylonSix ?
